diff --git a/lib/targets/codegen.js b/lib/targets/codegen.js
index e338aee4dcc18c7f53a7d2d7c084cdadf4fa66b1..87f315d4180d91a4d0a8c2eb01a5390a5e9244f7 100644
--- a/lib/targets/codegen.js
+++ b/lib/targets/codegen.js
@@ -31,7 +31,7 @@ async function build({
     await (0, _spawn.spawn)('npx', ['react-native', 'codegen'], {
       stdio: 'ignore'
     });
-    (0, _patchCodegen.patchCodegen)(root, packageJson, report);
+    //(0, _patchCodegen.patchCodegen)(root, packageJson, report);
     report.success('Generated native code with codegen');
   } catch (e) {
     if (e != null && typeof e === 'object') {
